src/iocore/cache/Cache.cc

```diff
@@ -43,6 +45,9 @@
 #include <system_error>
 #include <filesystem>
 
+#define SCAN_BUF_SIZE              RECOVERY_SIZE
+#define SCAN_WRITER_LOCK_MAX_RETRY 5
```

Comment:
[nitpick] `constexpr` is preferred.

Reply:
I will make a note of it - I think it'd be good to change these all at once over the whole subsystem for consistency.